arduino-ide-extension/src/node/monitor-manager.ts

```diff
@@ -204,41 +199,40 @@ export class MonitorManager extends CoreClientAware {
    * Notifies the monitor service of that board/port combination
    * that an upload process started on that exact board/port combination.
    * @param fqbn the FQBN of the board connected to port
-   * @param port port to monitor
+   * @param beforePort port to monitor
    * @returns a Status object to know if the process has been
    * started or if there have been errors.
    */
   async notifyUploadFinished(
-    fqbn?: string | undefined,
-    port?: Port
+    fqbn: string | undefined,
+    beforePort: PortIdentifier,
+    afterPort: PortIdentifier
```

Comment:
will this param be used for something?

Reply:
https://github.com/arduino/arduino-ide/pull/2165/commits/76d46d40b58c9896a583fe833f4f4b21f0b7b4c5